src/jit/unwindamd64.cpp

```diff
@@ -532,8 +785,12 @@ void Compiler::unwindEmitFuncHelper(FuncInfoDsc* func, void* pHotCode, void* pCo
 {
     UNATIVE_OFFSET startOffset;
     UNATIVE_OFFSET endOffset;
-    DWORD unwindCodeBytes;
-    BYTE* pUnwindBlock;
+    DWORD unwindCodeBytes = 0;
+    BYTE* pUnwindBlock = nullptr;
+#ifdef UNIX_AMD64_ABI
+    DWORD cfiCodeBytes = 0;
+    BYTE* pCfiBlock = nullptr;
```

Comment:
Why split these out for CFI? Why not just reuse `unwindCodeBytes` and `pUnwindCodeBlock`? This would also let you remove the conditional at the end of the function.